CODEOWNERS

```diff
@@ -1 +1,2 @@
-
+# Brian Barr and Robert Reilly defined as Global Code Owners
+@tazitoo @rreilly10 
```

Comment:
This doesn't seem to match [the format required by GitHub](https://github.blog/2017-07-06-introducing-code-owners/).  I think you need a `*` and some whitespace first.

Reply:
updated per the syntax of https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners